<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseConfigTool</class>
 <widget class="QMainWindow" name="MouseConfigTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MouseConfigTool</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="getCurrentPowerGroup">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>590</y>
      <width>150</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>获取当前电量</string>
    </property>
    <widget class="QPushButton" name="getCurrentPowerBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>当前电量</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentPowerText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前电量 Text</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="usbConfigGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>200</height>
     </size>
    </property>
    <property name="title">
     <string>USB 配置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HID 设备</string>
     </property>
    </widget>
    <widget class="QPushButton" name="on_offBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboHIDBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>VID：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VIDLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PIDLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="usbUpdateGroup">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>420</width>
      <height>102</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>USB 升级</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="2">
      <widget class="QPushButton" name="selectHexFileBtn">
       <property name="text">
        <string>选择升级文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>升级</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="updateHexName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="filePathText">
       <property name="text">
        <string>文件路径：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="getCurrentDPIGroup">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>150</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>获取当前 DPI</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="getCurrentDPIModeBtn">
       <property name="text">
        <string>当前 DPI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentDPIText">
       <property name="text">
        <string>当前 DPI Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="getCurrentLEDGroup">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>150</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>获取当前灯效</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="getCurrentRGBModeBtn">
       <property name="text">
        <string>当前灯效</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentLEDText">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>当前灯效 Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="getDriverModeGroup">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>176</width>
      <height>120</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>120</height>
     </size>
    </property>
    <property name="title">
     <string>获取当前 Driver 模式</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="getCurrentDeviceModeBtn">
       <property name="text">
        <string>当前模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentModeText">
       <property name="text">
        <string>当前模式 Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="getMacroKeyGroup">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>120</height>
     </size>
    </property>
    <property name="title">
     <string>获取按键宏</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="getMultiKeyBtn">
       <property name="text">
        <string>当前侧键</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="marcoKey1Text">
       <property name="text">
        <string>侧键一：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="marcoKey2Text">
       <property name="text">
        <string>侧键二：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="marcroKeyGroup">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>600</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string>设置按键宏</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="setMultiKeyBtn">
       <property name="text">
        <string>按键宏</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="setDriverModeGroup">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string>设置 Driver 模式</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="setConfigModeBtn">
       <property name="text">
        <string>配置模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setNormalModeBtn">
       <property name="text">
        <string>正常模式</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="currentDPIGroup">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>150</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>300</height>
     </size>
    </property>
    <property name="title">
     <string>当前 DPI</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="DPIMode1Btn">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DPIMode2Btn">
       <property name="text">
        <string>800</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DPIMode3Btn">
       <property name="text">
        <string>1600</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DPIMode4Btn">
       <property name="text">
        <string>4000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="currentLEDGroup">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>150</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>300</height>
     </size>
    </property>
    <property name="title">
     <string>当前灯效</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="RGBMode4Btn">
       <property name="text">
        <string>跑马灯</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RGBMode3Btn">
       <property name="text">
        <string>呼吸灯</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RGBMode2Btn">
       <property name="text">
        <string>循环灯</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RGBMode1Btn">
       <property name="text">
        <string>无灯效</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="wpiLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mouseBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
